test/unit/lib/plugins/aws/package/compile/events/cognito-user-pool.test.js

```diff
@@ -802,4 +963,426 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
       ).to.equal('AWS::Lambda::Permission');
     });
   });
+
+  describe('Custom Sender Sources', () => {
```

Comment:
let's put all new tests based on `runServerless` in a separate `describe`

Reply:
Sounds great!